format-specs/spatial-type-guidance.md

```diff
@@ -0,0 +1,328 @@
+of storing those values alongside each element); however, many older type systems
+(e.g., PostgreSQL) only have the opportunity to store a type name or identifier.
+This requires that all information required to interpret a value lives with each element
+(which, depending on the type, can involve a lot of repetition).
+
+If the format allows, spatial data types should be parameterized with a Coordinate
+Reference System to avoid unnecessarily repeating this information. This also allows
+better interopearability with other spatial data types that store the Coordinate
+Reference System in this way (e.g.,
+[GeoPandas](#),
+[R/sf](https://r-spatial.github.io/sf/reference/st_crs.html),
+[GeoArrow](https://geoarrow.org/extension-types.html#extension-metadata),
+[GeoParquet](https://github.com/opengeospatial/geoparquet/blob/v1.1.0/format-specs/geoparquet.md#crs)
+).
+
+There are cases where a format does not allow a type to be parameterized (i.e., a name
+or identifier is all that is possible). PostgreSQL is an example where this is the case:
+as a consumer of PostgreSQL using a non-spatial client, all the information you have
+is the name or ID of the type ("geometry" or "geography"). The Spatial Type implementation
+for PostgreSQL, PostGIS, uses two mechanisms to make the CRS of a column known:
+
+- It stores a CRS identifier (an integer) with every element that refers to a
+  `spatial_ref_sys` table that contains an extended string definition of that CRS.
+- It keeps a record of all
+  [geometry columns as a VIEW](https://postgis.net/docs/using_postgis_dbmanagement.html#geometry_columns)
+  and includes CRS identifier (an integer) that refers to the `spatial_ref_sys` table.
+
+(These mechanisms are also formalized in the
+[OGC Simple Feature Access – Part 2: SQL Option](https://www.ogc.org/publications/standard/sfs/))
+
+For clients reading a table directly, it is possible to look up the type-level CRS
+by querying the `geometry_columns` and `spatial_ref_sys` tables; however, for clients
+executing a complex query, it is necessary to inspect the integer that is sent alongside
+each encoded geometry in the result (which is almost always the exact same value for
+every element in the result). An important consequence of this workaround is that
+once the context of the original database connection is lost, there is no mechanism
+to obtain the full CRS definition to pass on to another Spatial Type[^3].
+
+[^3]: To mitigate this problem, PostGIS does use generally consistent identifiers
+for common CRSes such that it is usually possible to guess the CRS based on the
+integer identifier.
+
+## Coordinate Reference Systems
+
+A Coordinate Reference System (CRS) is can be conceptualized as a "unit"
+for the combinations of x, y, z, and/or m values of which geometries are
+comprised. Just as a value of "5 meters" has physical meaning whereas
+the value "5" does not, geometries with a coordinate reference system have
+physical meaning.
```

Comment:
Speaking about units doesn't give much justice to the concept of CRS. Perhaps quicly mentions that it encodes if the coordinates are geographic (longitude / latitude) or projected, and the underlying reference ellipsoid